src/outlet/ticketStorage.ts

```diff
@@ -226,28 +226,30 @@ export class TicketStorage {
 		return obj
 	}
 
-	private async updateOrInsertTicket(collectionHash: string, tokenRecord: StoredTicketRecord) {
-		const collectionTickets = this.ticketCollections[collectionHash] ?? []
-
-		for (const [index, ticket] of Object.entries(collectionTickets)) {
-			// TODO: Can be removed with multi-outlet
-			// Backward compatibility with old data
-			if (!ticket.tokenId || !ticket.type) {
-				ticket.type = ticket.type ?? 'asn'
-				ticket.tokenId = this.getUniqueTokenId(await this.decodeTokenData(ticket.type, ticket.token))
-			}
-
-			if (ticket.tokenId === tokenRecord.tokenId) {
-				collectionTickets[index] = tokenRecord
-				this.ticketCollections[collectionHash] = collectionTickets
-				this.storeTickets()
-				return
+	private async updateOrInsertTicket(collectionHash: string, tokenRecord: StoredTicketRecord): Promise<boolean> {
+		{
```

Comment:
hi, @nicktaras , why do we need extra brackets?

Reply:
Thanks for spotting this. It was an accidental merge conflict when aligning yours and Miccy's changes.